Implementation of Microsoft Graph Connector (#1)
ballerina-platform/module-microsoftgraph: src/msonedrive/Module.md

> Connects to Microsoft OneDrive and provides information on the files which have been stored on OneDrive.

Reviewer: ```suggestion
This module allows users to connect to Microsoft OneDrive and provides information on the files, which have been stored on OneDrive.
```

Author: Done

# Module Overview
This module contains operations for accessing the items stored in OneDrive.

## Compatibility
|                     |    Version     |
|:-------------------:|:--------------:|
| Ballerina Language  | 1.2.0   |
| Microsoftgraph REST API | v1.0          |

## Supported Operations

> - Get item from root directory

Reviewer: ```suggestion
- Get an item from the root directory
```

Author: Updated the text as follows.
"Get an item from the root directory"

> - Get item from a non-root directory

Reviewer: ```suggestion
- Get an item from a non-root directory
```

Author: Done

## Sample Application

> Please refer the sample application on the README.

Reviewer 2: Shall we add some code snippets of supported API calls? with a small description. like in. This module description appears in central. 
1. https://github.com/ballerina-platform/module-slack/blob/master/src/slack/Module.md
2. https://github.com/ballerina-platform/module-googlespreadsheet/blob/master/src/googleapis.sheets4/Module.md

Author: Ok, Done

> Please refer the sample application on the README.

Reviewer: ```suggestion
Refer the sample application on the README.
```
Can we link to the README file?

Author: Based on the comment from Danesh, I will add some code snippets of supported API calls here. Hence I will remove the highlighted sentence.
